editors/nanorc/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        scopatz nanorc 2020.10.10
+
+checksums           rmd160  907c5f9cac9a3c8bafdf8ec8f60d6c4028a0b880 \
+                    sha256  33a90657defa32ca7b1e91e0a67b67a4a63751973f7e70c1c638746162c1b3ea \
+                    size    77966
+
+homepage            https://github.com/scopatz/nanorc
```

Comment:
`homepage` does not need to specified as it will be automatically set by `github.setup` above.